x-pack/test/api_integration/deployment_agnostic/README.md

```diff
@@ -59,17 +65,19 @@ x-pack/test/<my_own_api_integration_folder>
 │  │     ├─ <serverless_project>.serverless.config.ts // e.g., oblt.serverless.config.ts
 │  ├─ ftr_provider_context.d.ts  // with types of services from './services'
 │  ├─ services
-│     ├─ index.ts // only services from 'x-pack/test/api_integration/deployment_agnostic/services'
+│     ├─ index.ts // only services from 'x-pack/platform/test/api_integration_deployment_agnostic/services'
```

Comment:
I think this change was missing in the original PR - might be worth to update in one of the next PRs.